src/test/java/io/pravega/connector/boomi/PravegaBrowserTest.java

```diff
@@ -149,9 +149,9 @@ public void testTestConnectorSuccess() throws Exception {
     @Test
     public void testTestNautilusConnector() throws Exception {
         String home = System.getProperty("user.home");
-        String jsonData = new String(Files.readAllBytes(Paths.get(home + "/keycloak.json")));
+        String jsonData = new String(Files.readAllBytes(Paths.get(home + "/keycloak-shield.json")));
```

Comment:
why are these things changed (json file, scope, stream)?  these should be fixed for the test and the tester must configure the SDP cluster to match expected scope, stream, etc.  we should not change these in the repository simply because they are different on our local machine - if they will vary that much, then export them to a properties file.